projects/admin.py

```diff
@@ -30,13 +29,11 @@ class PublicationAdmin(ProjectFields, admin.ModelAdmin):
         "doi",
         "link",
         "added_by_username",
-        "entry_created_date",
```

Comment:
Why is this dropped? I think we should keep track of this still.

Reply:
You are right, we should keep track of this. So I added this field in `PublicationSource` model so we can keep track of each source adding the publication

Reply:
Added migrations with data migration logic.